src/style.css

```diff
@@ -2,0 +1,24 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. It's recommended to add margin only to the top or bottom to avoid potential margin collapse issues. In this case, you have added a right margin to the `.stars__star` class.